### ISSUE TITLE
Add back button on project page to go back to wherever you came from

### ISSUE THREAD

**Issue by brandonhaslegs** [radicle-dev/radicle-upstream#1891]
Add back button on project page to go back to wherever you came from.

Things to think about:
- The back button doesn't always go back to the project list on the profile
- What happens when you get to a project from the search bar? (probably just do not show this back button)
- Should use the same styling as the back button when you click into see single commits
- Consider redesigning this back button it's kinda just an icon… maybe could be more obvious? Maybe not…

![image](https://user-images.githubusercontent.com/4406983/118471178-c1434f00-b707-11eb-92d2-92bbda9f189f.png)

**Comment by juliendonck** [comment]
@rudolfs this is possible now with the new router, right?

**Comment by rudolfs** [comment]
Yes.

**Comment by rudolfs** [comment]
How about this?

<img width="1552" alt="Screenshot 2021-10-15 at 15 47 23" src="https://user-images.githubusercontent.com/158411/137497753-9b21ad91-b854-4834-b0fd-732592314097.png">

**Comment by rudolfs** [comment]
In the current layout there's a bit less space:
<img width="1552" alt="Screenshot 2021-12-01 at 11 47 07" src="https://user-images.githubusercontent.com/158411/144220880-222718db-44df-431b-b430-32e0cd343424.png">

Should the back button still go on the left side of the project title?

**Comment by brandonhaslegs** [comment]
Yes. It's not to different from the commits back button. Bit tighter but i think it's fine.

<img width="306" alt="image" src="https://user-images.githubusercontent.com/4406983/144318935-5e2feae2-d78d-492c-b6c9-22c33ad18208.png">

**Comment by rudolfs** [comment]
We tried: https://github.com/radicle-dev/radicle-upstream/pull/2653#issuecomment-984950167.
